### ISSUE TITLE
[Feature request] Different icons for different file types

### ISSUE THREAD

**Issue by thebluedev** [aleksey-hoffman/sigma-file-manager#16]
## Feature request:
Right now most files without thumbnails have the same icon, 
![image](https://user-images.githubusercontent.com/73629408/119523480-69a58300-bd9a-11eb-97b8-8948372d3ade.png)

I would be better if they had different icons according to their extensions

**Comment by mattm-malone** [comment]
+1, I'd like to see something for this.

It looks like there's some differentiating between some file extensions, but not all (as you pointed out). I think it would be good to add more of these, but also have different icons for different extension categories.
![image](https://user-images.githubusercontent.com/31638037/119563444-154cd400-bd97-11eb-9150-8d354ce2f440.png)

Currently the icons are from Google's Material UI icons, for example: https://fonts.google.com/icons?icon.query=insert
This might be a bit restrictive, but that might be a good thing. For example all text files (.txt, .json, .js, .docx, etc.) would have a more "document" looking icon like so: https://fonts.google.com/icons?icon.query=document

I'm also wondering if a more vscode-like approach would work, where each file extension has it's own icon. Something like these? https://materialdesignicons.com/

**Comment by thebluedev** [comment]
Yes I agree, a more "vs code" like approach would be great! With the ability to customize too!

**Comment by aleksey-hoffman** [comment]
Hey bro, sorry for the late reply 😅
I'm releasing v1.5 update tomorrow, I'm looking through old issues to see if there's something else I could squeeze in.

I'm adding extensions (plugins) feature in v1.6. You will be able to add custom icons through extensions yourself
